src/Crud/Controller/Create.php

```diff
@@ -34,22 +36,32 @@ final class Create implements CrudControllerInterface
     /** @var EventDispatcherInterface */
     private $dispatcher;
 
-    public function __construct(DataStoreInterface $dataStore, FormFactoryInterface $formFactory, EventDispatcherInterface $dispatcher)
+    /** @var AuthorizationCheckerInterface */
+    private $checker;
+
+    public function __construct(DataStoreInterface $dataStore, FormFactoryInterface $formFactory, EventDispatcherInterface $dispatcher, AuthorizationCheckerInterface $checker)
     {
         $this->dataStore = $dataStore;
         $this->formFactory = $formFactory;
         $this->dispatcher = $dispatcher;
+        $this->checker = $checker;
     }
 
     public function __invoke(Request $request): ApiResponse
     {
         // Metadata you can specify in routing definition
         $modelClass = $request->attributes->get(self::MODEL_ATTRIBUTE);
         $form = $request->attributes->get(self::FORM_ATTRIBUTE);
+        $securityCheck = $request->attributes->get(self::SECURITY_CHECK, null);
```

Comment:
![image](https://user-images.githubusercontent.com/14903820/54519117-0e7bd880-4966-11e9-92c1-78376f93e3f9.png)

Reply:
It's only used when you use it. Documentation is definitely missing. It will be my next move !